cognite_toolkit/cdf.py

```diff
@@ -205,43 +217,43 @@ def deploy(
         typer.Option(
             "--env",
             "-e",
-            help="Build environment to build for",
+            help="CDF project environment to build for. Defined in environments.yaml. Defaults to dev.",
         ),
     ] = "dev",
     interactive: Annotated[
         Optional[bool],
         typer.Option(
             "--interactive",
             "-i",
-            help="Whether to use interactive mode when deciding which modules to deploy",
+            help="Whether to use interactive mode when deciding which modules to deploy.",
         ),
     ] = False,
```

Comment:
I am uncertain whether we should include this in the beta release. Suggest removing it .